src/approval/index.ts

```diff
@@ -23,16 +23,15 @@ export async function request(argv: RequestArguments): Promise<number> {
 
   if (typeof stackArgs.ApprovedTemplateLocation === "string" && stackArgs.ApprovedTemplateLocation.length > 0) {
     const s3 = new S3();
-    const s3Args = await approvedTemplateVersionLocation(stackArgs.ApprovedTemplateLocation, stackArgs.Template, argv.argsfile);
+    const s3Args = await approvedTemplateVersionLocation(stackArgs.ApprovedTemplateLocation, stackArgs.Template, argv.argsfile, argv.environment!);
```

Comment:
why `!`?

Reply:
It has to do with the type definition in https://github.com/unbounce/iidy/blob/eb7ca78117a56d5d6dc227fbf0ec7b9e250f2615/src/cli.ts#L28-L35 It will always be set (because yargs has a default value for it), but I wasn't able to make the types happy when dropping the `?`.

Reply:
I originally had `argv.environment || 'development'`, but I didn't feel ok having the fallback value in more than one place (even though the `||` would never be used anyway 🤷‍♂️ )